packages/react-cookie/package.json

```
@@ -38,7 +38,7 @@
   "dependencies": {
     "@types/hoist-non-react-statics": "^3.0.1",
     "hoist-non-react-statics": "^3.0.0",
-    "universal-cookie": "^4.0.0"
+    "universal-cookie": "^4.0.4"
```

No need to change this considering it can be used with 4.0.0.
```suggestion
    "universal-cookie": "^4.0.0"
```